corehq/apps/groups/templates/groups/group_members.html

```diff
@@ -76,52 +76,43 @@
                 return;
             });
 
-            function success(name, id, gaEventLabel) {
+            function outcome(success, name, id, gaEventLabel) {
                 return function() {
-                    unsavedChanges[name] = false;
-                    $('.alert').removeClass('alert-error alert-info').addClass('alert-success');
-                    $('.alert')[0].innerHTML = name + " was successfully saved";
-                    $('.alert').show();
-                    $(id).find(':button').enableButton();
+                    var alertClass, message;
+                    if (success) {
+                        alertClass = 'alert-success';
+                        message = 'Successfully saved ' + name.toLowerCase() + '.';
+                        unsavedChanges[name] = false;
+                    }
```

Comment:
i think dimagimon will yell at you for some things in JS like not having a semi colon. airbnb has a good style guide that I try to stick to: https://github.com/airbnb/javascript. (they use `} else {` )